### ISSUE TITLE
Search within Text Note

### ISSUE THREAD

**Issue by Pratiwir** [federicoiosue/Omni-Notes#622]
I am using Omni-Notes version 5.5.0 on a tablet. I have a lot of text within some text notes and it would be very useful to be able to search within the text note itself, and maybe have a next function of there are several of the search word found. Highlighted found words would also be nice.

**Comment by federicoiosue** [comment]
I agree it would be nice to have. I'll keep your issue opened as reminder for future implementations, thanks!

**Comment by Abdullah-Hany** [comment]
i would be more than happy to be assigned to this issue

**Comment by federicoiosue** [comment]
@Abdullah-Hany feel free to fork the code and make a pull request of your work!

Thanks!

**Comment by semal259** [comment]
Can I still work on this?

**Comment by federicoiosue** [comment]
> Can I still work on this?

Of course!

**Comment by landry314** [comment]
4 years later! we need to be able to search within a note for a text string.
